vendor/assets/manifest.webapp

```diff
@@ -32,6 +32,18 @@
       "folder": "/",
       "index": "index.html",
       "public": false
+    },
+    "/": {
```

Comment:
Using /service-create does not force you to create a second file.

```
"/service-create": {
       "folder": "/",
       "index": "index.html",
       "public": false
  }
```

Means that requests to `my-accounts.bob.cozy.tools/service-create` will serve index.html at the root of my-accounts app. You can then use the javascript router to figure you shoud display the form.